### DISCUSSION TITLE
Paperless login problem.

### DISCUSSION THREAD

**Discussion by BartekRadek13** [tteck/Proxmox#2136] [Help Needed] [answered]
Hi I`ve problem with login to paperless after install admin:admin doesnt work.

**Comment by tteck** [comment]
Where did you read that `admin:admin` is the login credentials?

**Comment by BartekRadek13** [comment]
From google.
Default login is admin:admin via the webui, accessible at
http://SERVERIP:PORT More info at paperless-ngx. For convenience this
container provides an alias to perform administration management commands.


sob., 2 gru 2023 o 02:10 tteckster ***@***.***> napisał(a):

> Where did you read that admin:admin is the login credentials?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tteck/Proxmox/discussions/2136#discussioncomment-7735619>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXAUF2F7X6Z5IR4QBODRRDYHJ5ZNAVCNFSM6AAAAABADTPWO6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TOMZVGYYTS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pozdrawiam
Bartłomiej Radka
783 260 477

**Comment by tteck** [comment, marked as answer]
![Screenshot 2023-12-01 8 17 50 PM](https://github.com/tteck/Proxmox/assets/21342670/b44002c3-ba36-46d8-9533-b25d7d1fe2c8)

**Comment by Sputnik24** [comment]
I upgraded to 2.0.1 with the script and cannot login anymore. Any idea?

**Reply by tteck** [reply]
Do you have a login screen?

**Reply by Sputnik24** [reply]
Yes. It says wrong username/password.

**Reply by tteck** [reply]
And it matches what is in `cat paperless.creds`?

**Reply by Sputnik24** [reply]
Yes.

I also tried python3 manage.py migrate, ... createsuperuser with admin admin, but all didn't help. Login to DB using Adminer is working, btw.

**Comment by BartekRadek13** [comment]
I have found  cat paperless.cred and there was randomly generated password.

niedz., 3 gru 2023 o 14:31 Sputnik24 ***@***.***> napisał(a):

> Yes.
>
> I also tried python3 manage.py migrate, ... createsuperuser with admin
> admin, but all didn't help. Login to DB using Adminer is working, btw.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tteck/Proxmox/discussions/2136#discussioncomment-7743714>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXAUF2WAFDLAS3TBJS4FKTYHR5L5AVCNFSM6AAAAABADTPWO6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TONBTG4YTI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pozdrawiam
Bartłomiej Radka
783 260 477
